server/passport/index.js

```diff
@@ -11,14 +11,19 @@ const setAuthRouter = function(router) {
   // OAuth callback url
   router.get(
     "/api/slack/callback",
-    passport.authenticate("slack", { failureRedirect: "/login" }),
-    (req, res) => {
-      const { user } = req;
-      res.redirect("/");
-    }
+    passport.authenticate("slack", { 
+      failureRedirect: "/login",
+      successRedirect: '/'
+      })
```

Comment:
Added failureRedirect in case something goes wrong from slack or mongo db while generating session